.github/workflows/ci.yml

```diff
@@ -16,32 +17,35 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
   build:
+    # bypass the action if the PR is a draft
     if: github.event.pull_request.draft == false
     name: build
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
 
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
         id: npm-cache
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json', '.nvmrc') }}
 
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
+        id: dist
         with:
-          path: '**/build'
-          key: ${{ runner.os }}-build-${{ github.sha }}
+          path: '**/dist'
```

Comment:
dev: this was wrong all along as the build directory is called `dist` and not `build`

Reply:
Nice catch